### PR TITLE
Moved Google CSE to span width under header, set div to absolute to f…

### PR DESCRIPTION
**Pull request by RichieEscarez**
…ix scrolling, added new "whats k8s" topic, and CSS fixes for headers

Added redirect to cover the .../docs url
Added local copy of "what is k8s"
Add CSS to no code wrapping

This is the last one from me for GA (ill continue iterating and refining tomorrow but otherwise we should be GA ready)

@brendandburns @thockin @bgrant0607 @krousey

### CODE REVIEW

**Comment by brendandburns** [comment]
LGTM.

**Comment by brendandburns** [comment]
@bgrant0607 do you want to take a look at this before we merge?

**Comment by bgrant0607** [comment]
Looking

File: _includes/nav_v1.0.html [hunk 1]
```diff
@@ -294,9 +267,10 @@
 			<li>
 				<a href="{{ site.url }}/v1.0/docs/user-guide/node-selection/README.html" class="nav-link">Assigning Pods to Nodes</a>
 			</li>
-			<li>
+			<!-- not a usefull topic
```
Comment (bgrant0607) [inline _includes/nav_v1.0.html]: In the future, let's please delete such cruft rather than comment it out.

File: _includes/nav_v1.0.html [hunk 2]
```diff
@@ -343,7 +317,7 @@
 			<li class="navSubMenu3">
 				<a href="{{ site.url }}/v1.0/examples/nfs/README.html" class="nav-link">NFS Volumes</a>
 			</li>
-			<!--><li class="navSubMenu3">
+			<!--<li class="navSubMenu3">
```
Comment (bgrant0607) [inline _includes/nav_v1.0.html]: You're commenting out the item open but not the close.

File: v1.0/docs/whatisk8s.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+layout: docwithnav
+---
+
+# What is Kubernetes?
+
+Kubernetes is an open source container cluster orchestration platform that was started by Google in 2014.
```
Comment (bgrant0607) [inline v1.0/docs/whatisk8s.md]: This hasn't incorporated feedback from #11628.

**Comment by bgrant0607** [comment]
I'm going to merge and then fix it.

File: _includes/nav_v1.0.html (hunk 1 above)
Comment (RichieEscarez) [inline _includes/nav_v1.0.html]: removed them all since i was fixing the item below

File: _includes/nav_v1.0.html (hunk 2 above)
Comment (RichieEscarez) [inline _includes/nav_v1.0.html]: fixed (sorry, was result from merge conflict)

File: v1.0/docs/whatisk8s.md (hunk 3 above)
Comment (RichieEscarez) [inline v1.0/docs/whatisk8s.md]: incorporating now

Comment (RichieEscarez) [inline v1.0/docs/whatisk8s.md]: see fix when in already through #11628

**Pull request merged**